### ISSUE TITLE
Convert JSON Schema to RDF

### ISSUE THREAD

**Issue by jviotti** [Intelligence-AI/jsonschema#85]
More info an examples here: https://json-schema.slack.com/archives/C5CF75URH/p1718199219779449

**Comment by hoijui** [comment]
@jviotti, that seems to be a private link. can you maybe make the information public, too?
(if possible and not too much trouble, only)

**Comment by jviotti** [comment]
@hoijui Hm, the link doesn't resolve to me either. Not sure. But overall, there was not much discussion around it as far as I remember more than somebody asking for it with a specific use case in mind.

Is it something you are looking for? If so, what you share more about your use case?

**Comment by hoijui** [comment]
I am not sure how useful or feasible it would be. I deal with "RDF schema"s (more specifically OWL ontologies with SHACL shapes for validation), and I think the opposite would make more sense (generating JSON-Schema from an RDF ontology/vocabulary or from SHACL/ShEx shapes.
All the auto-conversion of formats in this are is a bit underdeveloped in my experience, though, so if JSON-Schema -> RDF ontology conversion would exist, it could at the very least be used in the mix for cross-validation of some sort.

... in short: Not much need from my side, just interest. Your comment was perfect to clear things up; thank you!

**Comment by jviotti** [comment]
Yeah, I think you touch on the core problem here: JSON Schema is very expressive, and SHACL => JSON Schema is always feasible, but JSON Schema => SHACL is often tricky. That said, it would be interesting to have at least some level of support.

**Comment by jviotti** [comment]
@hoijui This is now done! See https://www.sourcemeta.com/blog/json-schema-jsonld-interoperability/
